week2/409_longest_palindrome.py

```diff
@@ -0,0 +1,29 @@
+# https://leetcode.com/problems/longest-palindrome/description/
+from collections import defaultdict
+
+
+def longest_palindrome(self, s: str) -> int:
+    if len(s) == 1:
+        return 1
+
+        # create frequency dictionary
+    freq_dist = defaultdict(int)
+    for letter in s:
+        if letter in freq_dist:
```

Comment:
You used a default dict which is a nice use of collections, but why are you checking the existence?